drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -424,9 +424,8 @@ static int gpio_mcux_lpc_pm_action(const struct device *dev, enum pm_device_acti
 	case PM_DEVICE_ACTION_TURN_OFF:
 		break;
 	case PM_DEVICE_ACTION_TURN_ON:
-		const struct gpio_mcux_lpc_config *config = dev->config;
```

Comment:
just move this above line 419, at the start of the function. It's completely standard style in zephyr drivers to put driver device struct pointers at the start of the function. I don't know why it was put in this case statement in the first place.